### DISCUSSION TITLE
How to show "on this page" on mobile devices

### DISCUSSION THREAD

**Discussion by ouyangzhCN** [oleeskild/obsidian-digital-garden#220] [Q&A] [answered]
When browsing the note on a desktop computer, we can see an "on this page" feature that shows the list of headings or sections on the current page, which is very helpful. 

However, when I browse the same page on the smartphone, the "on this page" misses. I'm wondering what I can do to bring it back, or we can add that in the future?

Please see the following figures:
On PC:
![on PC](https://user-images.githubusercontent.com/93504895/223010616-24788cc9-661e-4ff1-ae24-ef4e987dbb1f.png)
on smartphone:
![On mobile device](https://user-images.githubusercontent.com/93504895/223010623-4a89bd85-17c4-4229-85e4-46deeba5d763.png)

thank you!

**Comment by oleeskild** [comment, marked as answer]
Currently I just haven't implemented a view for it on mobile yet. But I've had it on my todo list for a while to add this. I've created a new issue for it. Hopefully I'll be able to add it soon: #242

**Reply by ouyangzhCN** [reply]
Thank you! Looking forward to it!
